### PR TITLE
feat: add method to load file into database

### PR DESCRIPTION
**Pull request by nisargparikh69**
Closed: #1 

- if file name is passed with load argument it will parse and load into db

### CODE REVIEW

**Comment by newtoallofthis123** [comment]
Pretty cool! Thanks for the help @nisargparikh69!

**Comment by newtoallofthis123** [comment]
Checks out :)

File: src/commands.rs [hunk 1]
```diff
@@ -1,6 +1,10 @@
 use std::{io::Write, path::Path};
 
-use crate::{file::file_exists, utils::DisplayEnv};
+use crate::{
+    file::{self, file_exists},
+    utils::DisplayEnv,
+};
+use bunt::println as print;
```
Comment (newtoallofthis123) [inline src/commands.rs:7]: Should have done this earlier 😅

Comment (nisargparikh69) [reply src/commands.rs:7]: I have just done this in one file just to showcase to you , later you can do same in other files... its long words to type on each print with bunt

**Pull request merged**